specs/agents/metadata.md

```diff
@@ -37,10 +37,13 @@ if os == windows
   hostname = exec "cmd /c hostname"                   // or any equivalent *
```

Comment:
I wonder if the prose paragraph above should be revamped, given the collection is now about `host.name` as well.  The notes there about following Go's `os.Hostname()` to match beats is no longer the case.

Reply:
yes, having to report the FQDN usually means there might be a DNS query involved, so let's remove this.